cg/services/order_validation_service/workflows/fastq/models/sample.py

```diff
@@ -0,0 +1,13 @@
+from cg.models.orders.sample_base import PriorityEnum, SexEnum
+from cg.services.order_validation_service.models.sample import Sample
+
+
+class FastqSample(Sample):
+    concentration_ng_ul: float | None = None
+    elution_buffer: str
+    priority: PriorityEnum
+    quantity: int | None = None
+    require_qc_ok: bool
+    sex: SexEnum
+    source: str  # Required if no internal id
```

Comment:
There are no internal ids sent from the front end for fastq orders - I suspect the Atlas info might be outdated. Internal Ids are sent for existing samples/reruns which we do not support for fastq.

Reply:
So this means that source is optional or that it should not be an attribute to consider?

Reply:
I was mainly thinking of the comment you added. The source seems to be correctly type hinted 💪